app/scripts/controllers/accessEvent.js

```diff
@@ -18,7 +18,7 @@ angular.module('confRegistrationWebApp')
           reasonForRequest: reasonForRequest,
           permissionLevel: 'REQUESTED'
         }
-      }).success(function () {
+      }).then(function () {
         $modalInstance.close();
       }).error(function (data) {
```

Comment:
Ah. I thought I had searched for all those. Guess I need to do it again and double check. Good catch :)